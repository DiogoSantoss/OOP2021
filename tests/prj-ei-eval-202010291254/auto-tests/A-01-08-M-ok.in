1
app03.dat
6
2
0
4
10
2
0
