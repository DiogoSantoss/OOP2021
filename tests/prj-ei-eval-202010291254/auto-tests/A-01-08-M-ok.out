Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Ficheiro a abrir: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: E4|Filomena Figueiredo|Lisboa, Portugal|NORMAL|0|0
ER|Abdul Figueiredo|Casablanca, Morocco|NORMAL|0|0
H2O|Rohit Figueiredo|New Delhi, India|NORMAL|0|0
H2SO4|John Figueiredo|Wellington, New Zealand|NORMAL|0|0
O9|Hellen Figueiredo|San Francisco, CA, USA|NORMAL|0|0
R2|Jorge Figueiredo|Lisboa, Portugal|NORMAL|0|0
TT|Jorge Figueiredo|Lx, PT|NORMAL|0|0
Gestão de Clientes
1 - Mostrar cliente
2 - Mostrar clientes
3 - Registar cliente
4 - Activa/desactivar notificações de um produto
5 - Mostrar transacções do cliente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Número de dias a avançar: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Clientes
7 - Gestão de Fornecedores
8 - Gestão de Transacções
9 - Menu de Consultas
10 - Ver Saldo Global
0 - Sair
Escolha uma opção: 